index.html

```diff
@@ -12,20 +12,26 @@
 <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300;400;500;700;800&family=Poppins:wght@400;700&family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
 </head>
 <body>
+  <div id="menu-section" class="menu-section">
+          <button id="menu-toggle" class="menu-toggle"><i class="fa-solid fa-xmark"></i></button>
```

Comment:
- [ ] kindly increase the size of the "**X**" button and change its color to match the Figma design 

![image](https://user-images.githubusercontent.com/89282221/217261050-c484793e-cc6c-411f-80b5-39236624c81c.png)